src/components/input/progress-bar.tsx

```diff
@@ -89,7 +89,7 @@ export function ProgressBar({
       <span
         className={cn(
           `absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-[45%]
```

Comment:
Pulled the code locally to have a look and it seems like the `-translate-y-[45%]` is causing the offset issue. Changing it to `-translate-y-1/2` should do the trick.

Regarding the colour, I'm using `text-light-secondary dark:text-dark-secondary` for the topic text at the moment, so that should do the trick here too.